packages/@react-aria/utils/test/domHelpers.test.js

```diff
@@ -23,6 +24,40 @@ describe('getOwnerDocument', () => {
     expect(getOwnerDocument(div)).toBe(document);
   });
 
+  it('returns the document if object passed in does not have an ownerdocument', () => {
+    const div = document.createElement('div');
+    expect(getOwnerDocument(div)).toBe(document);
+  });
+
+  it('returns the document if nothing is passed in', () => {
+    expect(getOwnerDocument()).toBe(document);
+    expect(getOwnerDocument(null)).toBe(document);
+    expect(getOwnerDocument(undefined)).toBe(document);
+  });
+
+  it('returns the documentOwner that is not document', () => {
```

Comment:
let's remove this one, this isn't something that should be set and is supposed to be readonly https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument
this might be a fluke of jsdom that we can set it

Reply:
The test is pretending the object is a jsdom object. I was trying to create tests that would demonstrate the correct behavior without the complexities of using iframes. I'll remove them.